Makefile

```diff
@@ -113,7 +113,7 @@ test_rpmbuild_cockpit-composer: buildrpm_image
 	sudo docker run --rm --name buildrpm -v `pwd`:/welder welder/buildrpm:latest make cockpit-composer-rpm cockpit-composer-srpm
 
 local-clean:
-	rm -rf test/images tmp cockpit-composer.spec cockpit-composer*.rpm welder-web*.tar.gz
+	rm -rf test/images tmp cockpit-composer.spec cockpit-composer*.rpm welder-web*.tar.gz test/end-to-end/node_modules test/end-to-end/wdio_report
```

Comment:
I would suggest to not kill node_modules. This is very expensive to download, and just annoying if you have to run several tests in a row.

Reply:
@martinpitt, if we keep node_modules and copy it into "machine", after npm install, the wdio (webdriver.io) cannot be run. (that's a bug of webdriver.io). So I have to clear it and re-install all dependencies.
Before, I always removed node_modules manually, not convenience.

Reply:
Eww, how ugly -- something writes into `node_modules/` during running the tests?? Could you please add a comment with a link to the bug report? This is outrageously annoying for local development.

I suggest a better workaround, like removing only the newly created file(s) which break stuff. Or, if it's hard to determine these reliably, copy test/end-to-end/node_modules to test/end-to-end/node_modules.clean after npm install, and copy back node_modules.clean to node_modules here.